qase-pytest/src/qaseio/pytest/testops.py

```diff
@@ -155,13 +156,14 @@ def _load_run(self):
                 return True
             return False
 
-    def _create_run(self, cases=[]):
+    def _create_run(self, plan_id=None, cases=[]):
         api_runs = RunsApi(self.client)
         result = api_runs.create_run(
             code=self.project_code,
             run_create=RunCreate(
                 title="Automated Run {}".format(str(datetime.now())),
                 cases=cases,
+                plan_id=int(plan_id),
```

Comment:
If we ignore the `--qase-to-plan` flag, we get an error of the following type:

`INTERNALERROR>   File "/qase-python/qase-pytest/src/qaseio/pytest/testops.py", line 166, in _create_run`
`INTERNALERROR>     plan_id=int(plan_id),`
`INTERNALERROR> TypeError: int() argument must be a string, a bytes-like object or a real number, not 'NoneType'`

An attempt will be made to cast a variable with the value None to int.

Reply:
Fixed